extensions/objects/protractor.js

```diff
@@ -242,6 +242,8 @@ var SkillSelector = function(elem) {
     setValue: async function(skillDescription) {
       await elem.element(
         by.css('.protractor-test-skill-name-input')).sendKeys(skillDescription);
+      await waitFor.visibilityOf(elem.element(
+        by.css('.protractor-test-rte-skill-selector-item')));
```

Comment:
Thanks for adding this!

Reply:
No problem... This was being recognised as a flake. Maybe it might not happen anymore 😛 . Fingers crosses 🤞 .